### DISCUSSION TITLE
EQ and balance?

### DISCUSSION THREAD

**Discussion by jmiller927** [micro-nova/AmpliPi#187] [General] [answered]
Hey guys, I'm considering AmpliPi for my home. Really dig that it's open source. But I have a question that I didn't see answered anywhere: Does AmpliPi give any left/right balance control and equalizer control (or even just simple bass/treble control) for the amplified output zones? For a multi-room setup, where each room has different acoustics and maybe different speakers, this is really important.

Similar amps in this price range seem to have these controls (e.g. [Monoprice 10761 6-zone amplifier](https://www.monoprice.com/product?p_id=10761), [Dayton Audio DAX88](https://www.daytonaudio.com/product/1746/dax88-8-source-8-zone-distributed-whole-house-audio-system-amplifier) and [DAX66](https://www.daytonaudio.com/product/1252/dax66-6-source-6-zone-distributed-audio-system)), but I couldn't find it discussed anywhere for AmpliPi. But maybe I'm missing something. Thanks.

**Comment by gorski123** [comment, marked as answer]
Hey jmiller927, thanks for your interest! So good question. Currently, the  AmpliPi hardware supports left/right balance, but it is not currently supported by software. For EQ/Treble/Bass there is potential to do this adjustment using ALSA per digital input, however in the current version there is no support for per zone EQ. At the moment one of our engineers is working on volume-level equalization interface across zone and audio sources in #186. This would allow you to set volume offsets between channels, zones, and sources which will help with volume level differences between rooms/speakers.

**Comment by jmiller927** [comment]
> in the current version there is no support for per zone EQ.

Is this a software or hardware limitation?

**Reply by gorski123** [reply]
The volume control chips we are currently using only support wide-band attenuation, they do not allow for attenuating specific frequency ranges, although there may be substitute parts that would support this.

**Reply by jmiller927** [reply]
Aw, bummer. Please consider adding per zone EQ (or just bass/treble) in a future hardware version. This is a must-have for many. Otherwise looks like a decent product.

**Comment by jonathanslenders** [comment]
@gorski123: I'm also struggling with the lack of an equalizer. In my dining room, I'm transitioning from a pair of Sonos Ones which sounded reasonably good, to a pair of Monitor Audio Bronze 50 speakers with the amplipi that I received some time back. I'm struggling to get a sound that matches the Sonos quality. I think, but I'm really not sure honestly, that I'm getting too much mid tones now.

Looking into it, it looks like Sonos has a "loudness" option to enhance certain frequencies when it's used at low volumes, this setting is on by default. I never adjusted the Sonos one built-in equalizer. So, maybe it is that "loudness" option that makes the difference. If so, It's maybe more about how audio should sound at low volume, rather then the specific room accoustics.

If at some point you can look into this, I'd appreciate that. Thanks!
